dm/syncer/error_test.go

```diff
@@ -208,3 +210,187 @@ func TestIsConnectionRefusedError(t *testing.T) {
 	isConnRefusedErr = isConnectionRefusedError(errors.New("connect: connection refused"))
 	require.True(t, isConnRefusedErr)
 }
+
+func TestGetDDLStatusFromTiDB(t *testing.T) {
+	var (
+		adminShowDDLJobsSQL1      string
```

Comment:
```suggestion
		adminShowDDLJobsSQL1      = fmt.Sprintf("ADMIN SHOW DDL JOBS 10")
```

and rest of variable initiation